beacon-chain/node/node.go

```diff
@@ -120,6 +120,9 @@ func New(cliCtx *cli.Context, opts ...Option) (*BeaconNode, error) {
 		return nil, err
 	}
 	prereqs.WarnIfPlatformNotSupported(cliCtx.Context)
+	if hasNetworkFlag(cliCtx) && cliCtx.IsSet(cmd.ChainConfigFileFlag.Name) {
+		return nil, errors.New("chain-config-file cannot be passed concurrently with network")
```

Comment:
If you use `fmt` then your message is resistant to flag renaming. 
```suggestion
		return nil, fmt.Errorf("%s cannot be passed concurrently with network", cmd.ChainConfigFileFlag.Name)
```